main.go

```diff
@@ -22,17 +23,43 @@ import (
 )
 
 var (
-	pass = color.FgGreen
-	skip = color.FgYellow
-	fail = color.FgHiRed
+	pass        = color.FgGreen
+	skip        = color.FgYellow
+	fail        = color.FgHiRed
+	gotestFlags = []string{
+		"-skipnotest",
+	}
+	parseFlags = make([]string,0)
+	args = make([]string,0)
+	ignore *bool
 )
 
 const paletteEnv = "GOTEST_PALETTE"
 
 func main() {
 	setPalette()
 	enableOnCI()
-	os.Exit(gotest(os.Args[1:]))
+
+	flagSet := flag.NewFlagSet("gotestFlags",flag.ContinueOnError)
+	ignore=flagSet.Bool("skipnotest", false, "skip packages with no test files")
```

Comment:
Move parseFlags here.

var parseFlags []string

I'd also name it to something like gotestFlags

Reply:
there is a gotestFlags variable already!